src/containers/BooksList.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Book from '../components/Book';
+import { removeBook } from '../actions';
+
+const BooksList = props => {
+  const { books } = props;
+  const handleRemoveBook = book => {
+    props.removeBook(book);
+  };
+
+  return (
+    <table>
+      <thead>
+        <tr>
+          <th>Book ID</th>
+          <th>Title</th>
+          <th>Category</th>
+        </tr>
+      </thead>
+      <tbody>
+        {books.map(book => (
+          <tr key={book.id}>
+            <Book book={book} removeBook={() => handleRemoveBook(book)} />
```

Comment:
Here you should just pass an instance of the function. For clarity, you could also change the props name to `handleRemoveBook`.
```suggestion
            <Book book={book} handleRemoveBook={handleRemoveBook} />
```